response_options/artemisTypeCap.jsonld

```diff
@@ -0,0 +1,53 @@
+{
+    "@context": "https://raw.githubusercontent.com/ReproNim/reproschema/1.0.0-rc1/contexts/generic",
+    "@id": "mriSoftwareValueConstraints",
+    "@type": "reproschema:ResponseOption",
+    "valueType": "xsd:integer",
+    "minValue": 0,
+    "maxValue": 6,
+    "multipleChoice": false,
+    "choices": [
+        {
+            "name": {
+                "en": "SPM"
+            },
```

Comment:
```suggestion
            "name": {
                "en": "Waveguard (ANT)"
            },
```

In the `response_options` files you have created you will need to replace the name for each response option with the list of choices we currently have in the table, like I did here.


In this case they go from 0  to 8:

0. Waveguard (ANT)
1. Electrocap
2. Quickcap (Neuroscan)
3. Easycap standard caps
4. Aegis Array
5. Emotiv
6. BrainProducts
7. Biosemi
8. TMSi



If you need to add more choices, then add a whole block like this one and then replace the name and value with the appropriate things.

```
        {
            "name": {
                "en": "name of the respone option"
            },
            "value":  integer_value
        }, 
```